pkg/util/kubernetes/apiserver/controllers.go

```diff
@@ -44,10 +44,6 @@ var controllerCatalog = map[controllerName]controllerFuncs{
 		},
 		startAutoscalersController,
 	},
-	servicesController: {
```

Comment:
we should be able to remove this, as this is a leftover from the last services PR. 
As we do not use them anywhere else and not as an own controller